test/jake/jakelib/migrate-test.js

```diff
@@ -543,6 +543,98 @@ test('migrate:down undos only the last run migration', (temp) => {
     });
 });
 
+test('list prints migrations both completed and pending', (temp) => {
+  const migrationFile1 = '001_create_books_table.js';
+  const migrationFile2 = '002_add_pages_column_to_books_table.js';
+
+  return assertExec(
+    `node ${KNEX} list \
+    --client=sqlite3 \
+    --connection=${temp}/db \
+    --migrations-directory=${temp}/migrations`
+  )
+    .then(({ stdout }) => {
+      assert.include(stdout, `No Completed Migration files Found.`);
+      assert.include(stdout, `No Pending Migration files Found.`);
+
+      fs.writeFileSync(
```

Comment:
using FileHelper with auto-cleanup is preferable for preparing files. see https://github.com/tgriesser/knex/blob/master/test/cli/migrate-make.spec.js